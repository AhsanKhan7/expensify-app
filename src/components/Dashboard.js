import React from 'react'
import ExpenseList from './expensesList'
import ExpenseListFilters from './ExpenseListFilters'

const dashboardPage = () => (
    <div>
        <ExpenseListFilters />
        <ExpenseList />
    </div>
)

export default dashboardPage