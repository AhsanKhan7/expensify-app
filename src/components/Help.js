import React from 'react'

const help = () => (
    <div>
        <p>this is help component</p>
    </div>
)

export default help