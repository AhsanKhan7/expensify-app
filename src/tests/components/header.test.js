import React from 'react'
import { shallow } from 'enzyme'
import { Header } from '../../components/Header'

test('should header run', () => {
    const Wrapper = shallow(<Header startLogout={() => {}}/>)
    expect(Wrapper).toMatchSnapshot()
})

test('should call logout when click button', () => {
    const startLogout = jest.fn()
    const Wrapper = shallow(<Header startLogout={startLogout}/>)
    Wrapper.find('button').simulate('click')
    expect(startLogout).toHaveBeenCalled()
})