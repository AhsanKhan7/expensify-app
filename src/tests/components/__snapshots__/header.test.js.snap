// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should header run 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/Add"
  >
    Add-Expence
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/help"
  >
    Help
  </NavLink>
  <button
    onClick={[Function]}
  >
    Log out
  </button>
</header>
`;
